scraper/dev/docker/Dockerfile.base

```diff
@@ -10,41 +12,52 @@ RUN useradd -d /home/seleuser -m seleuser
 RUN chown -R seleuser /home/seleuser
 RUN chgrp -R seleuser /home/seleuser
 
-RUN apt-get -y update
-RUN apt-get install -y -q software-properties-common wget
+RUN apt-get update -y && apt-get install -yq \
+    software-properties-common
 RUN add-apt-repository -y ppa:openjdk-r/ppa
-RUN wget -qO- https://deb.nodesource.com/setup_4.x | sudo bash -
-RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && apt-get update -y && apt-get install -y -q \
-  openjdk-8-jdk \
-  openjdk-8-jre \
-  nodejs \
-  x11vnc \
+RUN apt-get update -y && apt-get install -yq \
+    curl \
+    wget \
+    sudo -yq \
```

Comment:
You don't need to add `-yq` to every line, once for the `apt-get install` command is enough